frontend/src/metabase/common/hooks/use-question-list-query/use-question-list-query.unit.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import LoadingAndErrorWrapper from "metabase/components/LoadingAndErrorWrapper";
+import { createMockCard } from "metabase-types/api/mocks";
+import { setupCardsEndpoints } from "__support__/server-mocks";
+import {
+  renderWithProviders,
+  screen,
+  waitForElementToBeRemoved,
+} from "__support__/ui";
+import { useQuestionListQuery } from "./use-question-list-query";
+
+const TEST_CARD = createMockCard();
+
+const TestComponent = () => {
+  const { data = [], isLoading, error } = useQuestionListQuery();
```

Comment:
can't we use hook instead of this? I added a similar test to the component where the query hook is used and IDK what's the best place